types/knockout/test/index.ts

```diff
@@ -1,4 +1,4 @@
-
+/// <reference types="knockout.postbox" />
```

Comment:
There are tests that use knockout.postbox, so I had to choose whether to remove them or add the typings.